src/components/ui/multi-filter/filter-header.tsx

```diff
@@ -0,0 +1,29 @@
+import { ChevronLeft } from "lucide-react";
+import { useTranslation } from "react-i18next";
+
+import { Button } from "@/components/ui/button";
+
+function FilterHeader({
+  label,
+  onBack,
+}: {
+  label: string;
+  onBack: () => void;
+}) {
+  const { t } = useTranslation();
+  return (
+    <div className="flex items-center gap-2 px-3 py-2 border-b border-gray-200">
+      <Button
+        variant="ghost"
+        size="sm"
+        onClick={onBack}
+        className="h-6 w-6 p-0"
+      >
+        <ChevronLeft className="h-4 w-4" />
+      </Button>
+      <span className="text-sm font-medium">{t(label)}</span>
```

Comment:
we avoid apply `t` on the passed props... static analysis tools wouldn't be able to detect missing translations this way...